maquetado/scss/_titles.scss

```diff
@@ -0,0 +1,9 @@
+.title {
+  background: transparent url('assets/triangle.svg') no-repeat center left;
+  color: $black;
+  font-size: 24px;
+  height: $title-decoration-height;
+  padding-left: 52px;
+  padding-top: 35px;
```

Comment:
I feel you are using this just for the background. I think it's better if you use a [pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements), if you have any question, feel free to ask 😄

Reply:
Fixed with @henryzarza help
; )